src/index.js

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Kindly remove this & resolve it manually. It is bad to disable the eslint.